cardano-db/src/Cardano/Db/Run.hs

```diff
@@ -46,10 +49,11 @@ import           Database.Esqueleto.Experimental (SqlQuery)
 import           Database.Esqueleto.Internal.Internal (Mode (SELECT), SqlSelect, initialIdentState,
                    toRawSql)
 
+import           Data.Pool (Pool (..))
 import           Database.Persist.Postgresql (SqlBackend, openSimpleConn, withPostgresqlConn,
                    withPostgresqlPool)
```

Comment:
Unrelated to this pr, but it seems db-sync already uses connection pools in some places even though I don't find any apparent reason, since there are no parallel queries.

Reply:
Changed this to not use a connection pool.